exp/addons/controllers/clusterresourceset_controller_test.go

```diff
@@ -89,7 +88,7 @@ metadata:
 		}
 		By("Creating 2 ConfigMaps with ConfigMap in their data field")
```

Comment:
I'm guessing this needs to be updated to reflect that we are creating a ConfigMap and a Secret now.